source/_authors/crell.twig

```diff
@@ -0,0 +1,9 @@
+---
+title: Larry Garfield
+identifier: crell
+twitter: crell
+avatar: # TODO - REPLACE
+bio: Director of Developer Experience at Platform.sh. Collaboration advocate. Clean code advocate. Clean government advocate. Geek. #PSR8-compatible.
```

Comment:
@Crell I've drafted this for you, copying your twitter bio. Please suggest (or commit) what you prefer here (avatar too).